aos/closed_loop/ClosedLoopSimulation.ipynb

```diff
@@ -0,0 +1,448 @@
+{
```

Comment:
<p>Would be nice to see the PSSN/ FWHM plot here </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Added</p>